src/ipsis/course-merge/SelectedCourseUnmergeOfferingCollectionEntity.js

```diff
@@ -0,0 +1,28 @@
+import { Actions } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+export class SelectedCourseUnmergeOfferingCollectionEntity extends Entity {
```

Comment:
We have a separate `Selected` and regular `CollectionEntity` for merge, because in the merge process theres two separate pages. The first page where you select things and then the dialog where you select a target. For unmerge we don't need that separation so this would just be all of the unmerge offerings. 

Long story short could we name this `CourseUnmergeOfferingCollectionEntity`?

Reply:
I've deleted the file since it is now taken into account in Zak's PR https://github.com/BrightspaceHypermediaComponents/siren-sdk/pull/586

Reply:
`unmerge-course-offerings` is also added in Zak's PR. Is there a need for this PR anymore or can it be closed?

Reply:
It can be closed.